### DISCUSSION TITLE
How to use newer models like llama3 and phi3?

### DISCUSSION THREAD

**Discussion by cemremengu** [h2oai/h2o-llmstudio#704] [Q&A] [answered]
First of all thank you for this fantastic app. Works very smoothly. 

I was curious if there is a way to use newer models like llama3 or phi3 or  if you are planning to add them sometime in the future?

**Comment by pascal-pfeiffer** [comment, marked as answer]
The field for LLM Backbone is a free typing field. The dropdown is to get a few defaults already. Apart from that, basically all LLMs from huggingface can be used. For very exotic LLMs that are not in the transformers package (yet), activate `trust_remote_code`)

The two models you were mentioning are already fully supported. 

https://github.com/h2oai/h2o-llmstudio/assets/1069138/baec3586-1c08-4588-bf97-88775f9b997e

![image](https://github.com/h2oai/h2o-llmstudio/assets/1069138/825d0637-151f-44a3-94ee-1ca35a5dbd65)

**Reply by cemremengu** [reply]
Brilliant thank you!
